extensions/parquet/table/src/main/java/io/deephaven/parquet/table/TypeInfos.java

```diff
@@ -474,6 +474,7 @@ default Type createSchemaType(
                 builder = getBuilder(isRequired(columnDefinition), false, dataType);
                 isRepeating = false;
             }
+            instructions.getFieldId(columnDefinition.getName()).ifPresent(builder::id);
```

Comment:
You can skip it here, I am making the change and testing it as part of my PR  [here](https://github.com/deephaven/deephaven-core/pull/5989/files#diff-ba4d692529c4f1d53457e1944fbba983c7b8b40bd0655e95602a7b6704db2afd)

Reply:
The ability to write _Parquet_ field ids doesn't necessarily need to be tied into _Iceberg_'s usage of it. Given how simple it was here, I think we can leave it in?